indexer/services/comlink/src/controllers/api/v4/compliance-v2-controller.ts

```diff
@@ -276,6 +276,17 @@ router.post(
         complianceStatus,
         updatedAt,
       );
+      if (complianceStatus[0] !== complianceStatusFromDatabase &&
```

Comment:
when would `complianceStatus[0] !== complianceStatusFromDatabase` ever be equal?

I also don't think you can access `complianceStatus[0]`, because `complianceStatus` might have length 0

Reply:
Good catch!

Reply:
There are cases where it can be equal - like first_strike -> first_strike  or compliant->compliant